### ISSUE TITLE
IMU data comes in packets

### ISSUE THREAD

**Issue by frontw** [lgsvl/simulator#138]
We are working on car dynamics identification and use your simulator for this task as far as for other SDC project simulation tasks. 
For dynamics identification it's very important, how reliable is data, it should have constant rate and latency.
Here #137 I mentioned IMU data timestamp_sec is same for some consecutive messages. 
And then I made another test to see, how data actually comes.
I record bag (set /use_sim_time to false) and use rqt to display time, message actually came:
![Screenshot from 2019-04-18 13-19-06](https://user-images.githubusercontent.com/14126078/56355110-2e5d2100-61de-11e9-8121-b0c842365c6e.png)
And, as you can see, about 3-7 IMU msgs comes together in some sort of bunch. So, period is not 1/100.
I guess, there is some problem of buffering in part of Unity or rosbridge. I run simulator and Apollo on the same machine, so it is not network issue.  
You could compare IMU rate with tf, where publishing period is constant.
![Screenshot from 2019-04-18 13-21-49](https://user-images.githubusercontent.com/14126078/56355896-23a38b80-61e0-11e9-8e8a-ba3bf0b78ef4.png)
![Peek 2019-04-18 13-21](https://user-images.githubusercontent.com/14126078/56355374-cc50eb80-61de-11e9-84e7-d780f26f95e9.gif)

Thanks for the great project!

**Comment by martins-mozeiko** [comment]
We'll look into fixing this.

**Comment by daviduhm** [comment]
Hi @frontw,

Just wanted to update you that we have fixed this issue on our internal repository, and the fix will be available soon in a week or so after our weekly testing is done. In the meantime, here is the rqt graph showing IMU messages being published in a constant interval:

![ImuFix](https://user-images.githubusercontent.com/7874480/57114842-22a24c00-6d00-11e9-9042-9fb907f561e2.png)

**Comment by frontw** [comment]
Hi @daviduhm!
It’s super cool, thank you for this update! 
Looking forward to test it after publishing.
Btw, what was the reason of this issue?

**Comment by martins-mozeiko** [comment]
Our IMU implementation was tied to Unity physics update step and Unity does multiple physics steps as fast as it can during one frame. We did not notice this a problem because Apollo was driving just fine with original implementation. Now we changed code to produce IMU messages only when their corresponding values (velocity/acceleration) are supposed to be measured - which should give approximately  evenly spaced 100Hz.
